### DISCUSSION TITLE
Installation Help.

### DISCUSSION THREAD

**Discussion by kipmurray** [UI-Lovelace-Minimalist/UI#546] [Q&A] [answered]
I am going through the yaml-mode installation instructions and i get to a point where it says
`Download the files. Go to our repository on GitHub and download the code by pressing the green “Code” button in the top right corner and choose “Download ZIP”. Save the file on your computer and unpack it.`  I went to your GitHub page and downloaded the zip file as instructed, but the folders you reference in your instructions don't exist in the download file.  What am i doing wrong?

**Comment by basbruss** [comment]
Did you try it by following this guide? https://ui-lovelace-minimalist.github.io/UI/setup/download/

**Comment by kipmurray** [comment]
Thank you for responding @basbruss .

I did not follow those instructions, there is another set that I was using found here [https://ui-lovelace-minimalist.netlify.app/installation/yaml-mode](https://ui-lovelace-minimalist.netlify.app/installation/yaml-mode).

However, the page you linked to I had found at one point.  I also downloaded the included .zip file from that page.  When I open it up to explore through its contents, I noticed the `minimalist-templates` and `themes` folder still don't exist.

I'm still stuck.

**Comment by basbruss** [comment, marked as answer]
Since the first HACS-release it's packed in a different folder structure. So don't worry about the `minimalist-templates` and `themes` folders. When you place your unzipped package in `config/custom_components` as described on the wiki site everything should work fine after a reboot. The themes folder will be automatically added through a script 😉 

> I did not follow those instructions, there is another set that I was using found here https://ui-lovelace-minimalist.netlify.app/installation/yaml-mode.

That wiki is depreciated and outdated since UI Minimalist now uses HACS as main platform for releases.

**Comment by kipmurray** [comment]
@basbruss 

OMG, that makes WAY more sense.
Ok, I got it to work.  Thank you for that information.
Now to get to styling!  Thank you again for your help!
